Specs/DateTime/French/HolidayExtractor.json

```diff
@@ -142,4 +142,131 @@
       }
     ]
   }
-]
\ No newline at end of file
+]
```

Comment:
Almost right. Remove the closing and reopening square brackets ( ] [ ) and add a comma in line 144 to show the list continues with the new cases.